### ISSUE TITLE
Analytics: Add support for Parse.ly

### ISSUE THREAD

**Issue by tmmbecker** [swissspidy/web-stories-wp#9936]
Similar to #5935, it would be handy to have the ability to integrate with other analytics tool like Parse.ly. The standard Parse.ly plugin includes settings that allow for their pixel to be added to any CPT. 

![Screen Shot 2021-12-01 at 9 48 07 AM](https://user-images.githubusercontent.com/22805865/145082437-bf6abb79-5989-472c-afa9-2ea90f5d8559.png)

But when selecting the `web-story-page` post type, the JS isn't inserted.  

This was requested by a Newspack publisher.

**Comment by swissspidy** [comment]
If there's a dedicated Parse.ly WordPress plugin already, it would be that plugin's responsibility to add support for Web Stories. I highly recommend reaching out to them about it.

All they'd need to do is hook into the `web_stories_print_analytics` action.

Here's a quick plugin to demonstrate how it works:

https://gist.github.com/swissspidy/bdd58b2de495d5a691d015eb67736775

Of course we could add Parse.ly to the dropdown on the settings page as well, but most likely a publisher will have the official Parse.ly plugin installed already, so it makes more sense to add there.

**Comment by adamsilverstein** [comment]
@swissspidy based on https://www.parse.ly/help/integration/google-amp, would something like this work?

```php
add_action( 'web_stories_print_analytics', function() {
	  echo '<amp-analytics type="parsely"><script type="application/json">{"vars": {"apikey": "API_KEY"}}</script></amp-analytics>';
} );
```

**Comment by swissspidy** [comment]
That's what's in my Gist above :-) So, yes

**Comment by tmmbecker** [comment]
Thank you both! I've shared this with the Parse.ly team. In the meantime, I can use that hook to create a small plugin as a temporary workaround for this publisher. 🥂

**Comment by pauarge** [comment]
Hello!

This is Pau from the team that maintains the wp-parsely plugin. As per the issue above, we will be addressing this issue in an upcoming release.

**Comment by swissspidy** [comment]
@pauarge Awesome, thanks a lot for the update! I'll happily follow your progress and answer any questions you might have.

**Comment by pauarge** [comment]
@swissspidy Opened a PR if you wanna take a look: https://github.com/Parsely/wp-parsely/pull/602

**Comment by pauarge** [comment]
PR got merged and this will be made available on the next version of the plugin, 3.2.0.

**Comment by swissspidy** [comment]
Awesome, thanks a lot for the update!

I'm therefore closing this issue here :)
